qubes-split-gpg2.conf.example

```diff
@@ -0,0 +1,57 @@
+[DEFAULT]
```

Comment:
nitpick: isn't it a bit odd that section names are case sensitive while options aren't. Maybe it would make sense to have `DEFAULT` lower case for consistency. But maybe Python's default is already common?

Reply:
The `DEFAULT` section is special (for the parser), as it holds default values for all sections. You can change name of this special section (`default_section` argument), but IMO having this one upper case is clearer for its function.

Reply:
> [...] but IMO having this one upper case is clearer for its function

I would think `[default]` and `[client:foo]` is pretty clear. But I'm fine with keeping Python's default.